webapp/channels/src/components/thread_popout/thread_popout.tsx

```diff
@@ -55,6 +59,20 @@ export default function ThreadPopout() {
     useEffect(() => {
         if (teamId) {
             dispatch(getThread(currentUserId, teamId, postId));
+
+            // Since the statuses are fetched properly and timely by the thread viewer, manually fetch them here
+            async function fetchPostThread() {
+                const {data: posts} = await dispatch(getPostThread(postId, true));
+                if (posts) {
+                    const {data: result} = await dispatch(extractUserIdsAndMentionsFromPosts(Array.from(Object.values(posts.posts))));
+                    if (result) {
+                        await dispatch(getProfilesByIds(result.userIdsForProfilePoll));
+                        await dispatch(loadStatusesByIds(result.userIdsForStatusPoll));
```

Comment:
Testing locally, I noticed that these can cause a 400 error if no users need to be loaded, which seems to mostly happen if a user pops out one of their own threads. If `getProfilesByIds` errors out like that, the call to `loadStatusesByIds` would be skipped then.

Could you wrap these in a check to make sure they're not dispatched when those arrays are empty?